### ISSUE TITLE
Query parameters are not rendered to the href of Link

### ISSUE THREAD

**Issue by mike-marcacci** [FormidableLabs/redux-little-router#121]
If you create a `<Link />` element with either a string or object `href`, it renders the `<a />` tag without any of the query parameters. The `onClick` handler DOES redirect the browser to the correct place (with the query parameters), but omits it from the actual markup, breaking command-clicks, etc, and any automated traversing like search engines.

### Input
```es6
<div>
	<Link href='/pathname?foo=bar'>String</Link>
	<Link href={{pathname: '/pathname', query: {foo: 'bar'}}}>Query</Link>
	<Link href={{pathname: '/pathname', search: '?foo=bar'}}>Search</Link>
</div>
```

### Expected
```es6
<div>
	<a href='/pathname?foo=bar'>String</a>
	<a href='/pathname?foo=bar'>Query</a>
	<a href='/pathname?foo=bar'>Search</a>
</div>
```

### Actual
```es6
<div>
	<a href='/pathname'>String</a>
	<a href='/pathname'>Query</a>
	<a href='/pathname'>Search</a>
</div>
```

**Comment by tptee** [comment]
This is fixed in WIP PR #130! I had some bad tests stubs that resulted in false positives before 😬 I'll ping here on release.

**Comment by tptee** [comment]
Should be fixed in `v13.0.0`!
